src/style.css

```diff
@@ -0,0 +1,100 @@
+@font-face {
+  font-family: "Avenir";
+  src: url(fonts/Avenir.ttc);
+}
+
+* {
+  font-family: "Avenir", sans-serif;
+}
+
+body {
+  margin: 0;
+}
+
+.search {
+  margin: 20px 8px;
+}
+
+.search__block {
+  border: 1px solid #e1e7ed;
+  border-radius: 4px;
+  display: flex;
+  align-items: center;
+  background: #fff;
+  margin-bottom: 20px;
+}
+
+.search__block-big {
+  padding: 0 0 0 25px;
+}
+
+.search__block-small {
+  padding: 0 0 0 12px;
+  font-size: 14px;
+  font-weight: 400;
+}
+
+.label {
+  width: 100%;
+}
+
+.img {
+  margin-right: 18px;
+  margin-bottom: 1px;
+}
+
+.img-small {
+  margin-right: 10px;
+  margin-bottom: 1px;
+  max-width: 11px;
+}
+
+.input {
+  border: none;
+  outline: none;
+  width: 100%;
+  color: #3d4e61;
+  padding: 23px 0;
+  font-weight: 300;
+  font-size: 16px ;
+}
+
+.input::placeholder {
+  color: #3d4e61;
+}
+
+.input-small {
+  border: none;
+  outline: none;
+  width: 100%;
+  color: #3d4e61;
+  padding: 10px 0;
+  font-weight: 300;
+  font-size: 14px;
+}
```

Comment:
You have repeatable styles here. Follow the checklist to not repeat yourself.
<img width="992" alt="image" src="https://user-images.githubusercontent.com/99403198/216766248-477c6fed-b7c2-4190-9662-1b5041952c4d.png">